packages/storage/amplify_storage_s3_dart/lib/src/storage_s3_service/transfer/database/database_io.drift.dart

```diff
@@ -0,0 +1,23 @@
+// ignore_for_file: type=lint
```

Comment:
Note: This is generated. It should be hidden by default on future code reviews since `**/*.drift.dart` is added to `.gitattributes` in this PR.